docs/update.md

```diff
@@ -11,38 +11,13 @@ Distribution for PostgreSQL.
     Only the incremental update to a nearest minor version of the Operator is
     supported. To update to a newer version, which differs from the current
     version by more than one, make several incremental updates sequentially.
+    See [documentation archive](https://docs.percona.com/legacy-documentation/)
+    for documentation on previous versions of the Operator.
 
 The following steps will allow you to update the Operator to current version
 (use the name of your cluster instead of the `<cluster-name>` placeholder).
 
-1. Pause the cluster in order to stop all possible activities:
-
-    ``` {.bash data-prompt="$" }
-    $ kubectl patch perconapgcluster/<cluster-name> --type json -p '[{"op": "replace", "path": "/spec/pause", "value": true},{"op":"replace","path":"/spec/pgBouncer/size","value":0}]'
-    ```
-
-2. If you upgrade the Operator **from a version earlier than 1.1.0**, the
-following additional step is needed for the 1.0.0 → 1.1.0 upgrade.
-
-    ``` {.bash data-prompt="$" }
-    $ export CLUSTER=<cluster-name>
-    $ for user in postgres primaryuser $(kubectl get perconapgcluster/${CLUSTER} -o yaml | yq r - 'spec.user'); do args+="--from-literal=$user=$(kubectl get secret/${CLUSTER}-${user}-o yaml | yq r - 'data.password' | base64 --decode) "; done; eval kubectl create secret generic ${CLUSTER}-users "${args}"
-    ```
-
-    This command creates users’ secrets with existing passwords. Otherwise, new
-    secrets with autogenerated passwords will be created automatically,
-    so existing passwords will be overwritten.
-
-    !!! note
-
-        The `pgbouncer` user password is stored in encrypted form, and
-        therefore it is not included in the above command. If you know this
-        password and/or would like to update it, please add it as
-        `pgbouncer: base64encodednewpassword` to the resulted Secret manually.
-        Otherwise, this password needs no actions and will be overwritten by the
-        Operator during upgrade.
-
-3. Remove the old Operator and start the new Operator version:
+1. Remove the old Operator version:
```

Comment:
We should not remove old operator. We need to update the operator.
How to do it:
```
curl -s https://raw.githubusercontent.com/percona/percona-postgresql-operator/1.x/deploy/operator.yaml | yq w -d'4' -  "spec.template.spec.containers[0].env[0].value" "update" | kubectl apply -f -
kubectl wait --for=condition=Complete job/pgo-deploy --timeout=90s
```